src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +11,9 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        return userService.findByEmail(email).getEmail().equals(email)
+               && userService.findByEmail(email).getPassword().equals(password);
```

Comment:
Better call method findByEmail only once, not twice:
```suggestion
	User user = userService.findByEmail(email);
        return user.getEmail().equals(email) && user.getPassword().equals(password);
```